main/ble_utils.c

```diff
@@ -298,7 +298,11 @@ char *chartoa(ble_uuid_t uuid, const uint8_t *data, size_t len)
             p += sprintf(p, "%s,", data[i] & 0x01 ? "true" : "false");
             break;
         case CHAR_TYPE_2BIT:
-            p += sprintf(p, "%" PRIu8 ",", data[i] & 0x03);
+            for(; i < len; i++){
+                for(int8_t j = 6; j >= 0; j -= 2){
```

Comment:
Please make sure to follow the current coding style in terms of spaces and defining variables in the for loop.

Reply:
Is this from your implementation of this characteristic or a peripheral that passed the Bluetooth SIG certification?
I'd expect the LSB to be used first, especially since everything else in BLE is little endian